ci-operator/config/openshift/cluster-logging-operator/openshift-cluster-logging-operator-tech-preview.yaml

```diff
@@ -0,0 +1,175 @@
+base_images:
+  base:
+    name: "4.7"
+    namespace: ocp
+    tag: base
+  elasticsearch-operator-src:
+    name: tech-preview
+    namespace: logging
+    tag: elasticsearch-operator-src
+  elasticsearch-proxy:
+    name: tech-preview
+    namespace: logging
+    tag: elasticsearch-proxy
+  logging-curator5:
+    name: tech-preview
+    namespace: logging
+    tag: logging-curator5
+  logging-elasticsearch6:
+    name: tech-preview
+    namespace: logging
+    tag: logging-elasticsearch6
+  logging-eventrouter:
+    name: tech-preview
+    namespace: logging
+    tag: logging-eventrouter
+  logging-fluentd:
+    name: tech-preview
+    namespace: logging
+    tag: logging-fluentd
+  logging-kibana6:
+    name: tech-preview
+    namespace: logging
+    tag: logging-kibana6
+  oauth-proxy:
+    name: "4.7"
+    namespace: ocp
+    tag: oauth-proxy
+  ocp_4.6_cli:
+    name: "4.7"
+    namespace: ocp
+    tag: cli
+  ocp_4.7_cli:
+    name: "4.7"
+    namespace: ocp
+    tag: cli
+  ocp_builder_rhel-8-golang-1.15-openshift-4.6:
+    name: builder
+    namespace: ocp
+    tag: rhel-8-golang-1.15-openshift-4.6
+  ocp_builder_rhel-8-golang-1.15-openshift-4.7:
+    name: builder
+    namespace: ocp
+    tag: rhel-8-golang-1.15-openshift-4.7
+  ocp_builder_rhel-8-golang-openshift-4.6:
+    name: builder
+    namespace: ocp
+    tag: rhel-8-golang-openshift-4.6
+  openshift_release_golang-1.13:
+    name: release
+    namespace: openshift
+    tag: golang-1.13
+  openshift_release_golang-1.14:
+    name: release
+    namespace: openshift
+    tag: golang-1.14
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.14
+images:
+- from: base
+  inputs:
+    ocp_4.7_cli:
+      as:
+      - registry.ci.openshift.org/ocp/4.7:cli
+    ocp_builder_rhel-8-golang-1.15-openshift-4.7:
+      as:
+      - registry.ci.openshift.org/ocp/builder:rhel-8-golang-1.15-openshift-4.7
+  to: cluster-logging-operator
+- dockerfile_path: olm_deploy/operatorregistry/Dockerfile
+  from: base
+  to: cluster-logging-operator-registry
+- dockerfile_path: Dockerfile.e2e
+  from: elasticsearch-operator-src
+  to: cluster-logging-operator-e2e
+promotion:
+  name: tech-preview
+  namespace: logging
+resources:
+  '*':
+    requests:
+      cpu: 200m
+      memory: 200Mi
+tag_specification:
+  name: "4.7"
+  namespace: ocp
+tests:
+- artifact_dir: /tmp/artifacts
+  as: unit
+  commands: make test-unit
+  container:
+    from: src
+- artifact_dir: /tmp/artifacts
+  as: e2e-operator
+  steps:
+    cluster_profile: aws
+    env:
+      COMPUTE_NODE_TYPE: m5.4xlarge
+    test:
+    - as: test
+      cli: latest
+      commands: IMAGE_FORMAT= FAKE_HOME_DIR=/tmp/home LOG_DIR=/tmp/artifacts/logging-logs ARTIFACT_DIR=${ARTIFACT_DIR}/logging-artifacts make test-e2e-olm
+      dependencies:
+      - env: IMAGE_CLUSTER_LOGGING_OPERATOR_REGISTRYi
+        name: pipeline:cluster-logging-operator-registry
+      - env: IMAGE_CLUSTER_LOGGING_OPERATOR
```

Comment:
these env variables are supplied by whom? is it by `olm_deploy` scripts? https://github.com/openshift/cluster-logging-operator/blob/master/olm_deploy/scripts/catalog-deploy.sh#L4
We should change the image pull-spec here

Reply:
This is automagic of the CI which points back the images referenced above and get expanded